src/components/base/JTextArea/JTextAreaField.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+
+import React from 'react'
+import { Field } from 'react-final-form'
+
+import { ErrorMessage } from 'components/base/ErrorMessage/ErrorMessage'
+
+import { JTextArea } from './JTextArea'
+
+type FieldRenderProps = ExtractReturn<typeof Field>
+type Props = FieldRenderProps & {
+  ...HTMLTextAreaElement,
+}
+
+export function JTextAreaField(props: Props) {
+  const message = props.meta.visited ? String(props.meta.error) : ''
```

Comment:
I suggest we add this check as utility function, because it is typical form behaviour and we could need to change it all at once.

Reply:
Completely agree, It's more flexible way